object/merge.test.ts

```diff
@@ -0,0 +1,121 @@
+/* eslint-env jest */
+// @ts-ignore ambiguous import
+import merge from "./merge.ts";
+
+describe("merge", () => {
+  it("should merge with empty object", () => {
```

Comment:
Consider this:
"it should return the original object when merged with an empt object"
```
expect(merge(a, {})).toEqual(a);
expect(merge({}, a)).toEqual(a);
```